packages/tenant-process/src/utilities/errorMappers.ts

```diff
@@ -75,7 +75,7 @@ export const internalRevokeCertifiedAttributeErrorMapper = (
 ): number =>
   match(error.code)
     .with("tenantNotFound", () => HTTP_STATUS_NOT_FOUND)
-    .with("attributeNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .with("attributeNotFoundInTenant", () => HTTP_STATUS_NOT_FOUND)
```

Comment:
Since you are here, can you join these together on the same erro?

Reply:
Updated